standard/format_table

```diff
@@ -0,0 +1,91 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Format/Table
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Logic = require('Module:Logic')
+local Table = require('Module:Table')
+
+local TableFormatter = {}
+
+function TableFormatter.toLuaCode(tbl)
+	if type(tbl) ~= 'table' then
+		error('TableFormatter.toLuaCode needs a table as input')
+	end
+
+	if not next(tbl) then
+		return mw.html.create('pre')
+			:addClass('selectall')
+			:wikitext('{}')
+	end
```

Comment:
Why is this special case needed?

Reply:
it is the difference between
```lua
{}
```
and
```lua
{
}
```
(firs with the special case 2nd without